mediation-old/src/main/java/com/wso2telco/dep/mediator/ResponseHandler.java

```diff
@@ -282,16 +257,13 @@ public String walletPaymentResponseContext(MessageContext context, String jsonBo
 
         JSONObject jsonObj = new JSONObject(jsonBody);
         JSONObject objPay = jsonObj.getJSONObject("makePayment");
-        FileReader fileReader = new FileReader();
-        String file = CarbonUtils.getCarbonConfigDirPath() + File.separator
-                      + FileNames.MEDIATOR_CONF_FILE.getFileName();
-		Map<String, String> mediatorConfMap = fileReader.readPropertyFile(file);
-        String endUserId = objPay.get("endUserId").toString();
+
+				String endUserId = objPay.get("endUserId").toString();
```

Comment:
Formatting

Reply:
Please check the source code. this might be a presentaion problem in git hub.